android/versioned-abis/expoview-abi41_0_0/src/main/java/abi41_0_0/host/exp/exponent/VersionedUtils.java

```diff
@@ -5,33 +5,36 @@
 import android.app.Activity;
 import android.content.Context;
 import android.content.Intent;
-import android.content.pm.PackageManager;
 import android.net.Uri;
 import android.os.Build;
 import android.provider.Settings;
+import android.util.Log;
+
+import androidx.annotation.Nullable;
 
 import com.facebook.common.logging.FLog;
-import abi41_0_0.com.facebook.react.ReactInstanceManager;
-import abi41_0_0.com.facebook.react.ReactInstanceManagerBuilder;
-import abi41_0_0.com.facebook.react.common.LifecycleState;
-import abi41_0_0.com.facebook.react.common.ReactConstants;
-import abi41_0_0.com.facebook.react.devsupport.HMRClient;
-import abi41_0_0.com.facebook.react.packagerconnection.NotificationOnlyHandler;
-import abi41_0_0.com.facebook.react.packagerconnection.RequestHandler;
-import abi41_0_0.com.facebook.react.shell.MainReactPackage;
 
 import org.json.JSONObject;
 
+import java.lang.reflect.Field;
 import java.util.Collections;
 import java.util.HashMap;
 import java.util.Map;
 
-import androidx.annotation.Nullable;
+import abi41_0_0.com.facebook.react.ReactInstanceManager;
+import abi41_0_0.com.facebook.react.ReactInstanceManagerBuilder;
+import abi41_0_0.com.facebook.react.bridge.NativeModule;
+import abi41_0_0.com.facebook.react.bridge.ReactApplicationContext;
+import abi41_0_0.com.facebook.react.common.LifecycleState;
+import abi41_0_0.com.facebook.react.common.ReactConstants;
+import abi41_0_0.com.facebook.react.packagerconnection.NotificationOnlyHandler;
+import abi41_0_0.com.facebook.react.packagerconnection.RequestHandler;
+import abi41_0_0.com.facebook.react.shell.MainReactPackage;
+import abi41_0_0.host.exp.exponent.modules.api.reanimated.ReanimatedJSIModulePackage;
```

Comment:
Looks like there are much more imports than in other versions, but the code you added is the same 🤔